drivers/at86rf2xx/aes_spi.c

```diff
@@ -0,0 +1,432 @@
+/* Copyright (C) 2020 Otto-von-Guericke-Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_at86rf2xx
+ * @{
+ *
+ * @file
+ * @brief       Implementation of at86rf2xx SPI security module (AES)
+ *
+ * @author      Fabian Hüßler <fabian.huessler@ovgu.de>
+ * @}
+ */
+
+#include "xtimer.h"
+#include "periph/spi.h"
+#include "at86rf2xx_aes.h"
+
+#define ENABLE_DEBUG    (0)
+#include "debug.h"
+
+#define AES_DEBUG(...)  DEBUG("[at86rf2xx_aes_spi]: "__VA_ARGS__)
+
+#define AT86RF2XX_CMD_SRAM_READ             (0b00000000)
+#define AT86RF2XX_CMD_SRAM_WRITE            (0b01000000)
+
+static inline
+void at86rf2xx_spi_get_bus(const at86rf2xx_t *dev)
+{
+    spi_acquire(dev->params.spi, dev->params.cs_pin, SPI_MODE_0, dev->params.spi_clk);
+}
+
+static inline
+void at86rf2xx_spi_release_bus(const at86rf2xx_t *dev)
+{
+    spi_release(dev->params.spi);
+}
+
+static inline
+uint8_t _aes_status(at86rf2xx_t *dev)
```

Comment:
can't this be replaced with the `at86rf2xx_sram_xxx` functions (from 'at86rf2xx_internal.c`)?

Reply:
See https://github.com/RIOT-OS/RIOT/pull/14113/files#r433510445